src/components/Form.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {
+  Row, Form, Button, Col,
+} from 'react-bootstrap';
+
+const formStyle = [
+  {
+    width: '10.875rem',
+    height: '1.5rem',
+    margin: '1.813rem 32.75rem 1.188rem 0',
+    fontSize: '1.25rem',
+    fontWeight: 'Bold',
+    letterSpacing: '-0.18px',
+    color: '#888',
+  },
+];
+const FormComponent = () => (
+  <div className="form">
+    <h2 style={formStyle[0]}>ADD NEW BOOK</h2>
+    <Form>
+      <Row className="mb-4">
+        <Col xs={7} controlId="formGridCity">
+          <Form.Control placeholder="Book title" />
+        </Col>
+
+        <Form.Group as={Col} controlId="formGridState">
+          <Form.Select defaultValue="Category">
+            <option>Category</option>
+            <option>Action</option>
+            <option>Drama</option>
+            <option>Romance</option>
+          </Form.Select>
+        </Form.Group>
+        <Button as={Col} variant="primary" type="submit">
+          ADD BOOK
+        </Button>
+      </Row>
+    </Form>
```

Comment:
- [ ] You are missing another `form.Control` here, the one for the `author` of the book. A book has three attributes which are `title, author, and category`. Let's add it and everything should be working fine!🙂

Reply:
Thank you for your review I add the author input. I don't see the author case in the design because of that I didn't add it 😄 
 
![Capture](https://user-images.githubusercontent.com/80895497/135100340-5d80f71f-81ef-4979-ba13-57ca46326d6a.PNG)